samples/compose-browser/src/main/java/org/mozilla/samples/compose/browser/browser/BrowserScreen.kt

```diff
@@ -127,3 +148,29 @@ private fun Suggestions(
         onSuggestionClicked = { suggestion -> onSuggestionClicked(suggestion) }
     )
 }
+
+private data class ActionItemSpec(
+    val name: String,
+    val onClick: () -> Unit
+)
+
+@Composable
+private fun ActionMenu(
```

Comment:
Might be nice to have this as a separate file.

Reply:
My mistake, I wasn't expecting you to review so quickly.  I've moved the implementation to `BrowserToolbar` since `BrowserScreen` should not know how the toolbar is implemented.  Sure, I'll move it into a different file.